winterfell/views/login.pug

```diff
@@ -8,9 +8,10 @@ block emailText
   div.txt-container Login with your email
 
 block inputs
-  form
-    input.email(placeholder='Email Address', type='text')
-    input.pwd(placeholder='Password', type='password')
+  form(action='/api/auth/login' method='post')
```

Comment:
The previous code did not do any validation for login AFAICT, what validation would need to be done?